src/causalprog/distribution/family.py

```diff
@@ -67,13 +69,33 @@ def __init__(
         self._family = backend_family
         self._family_translator = backend_translator
 
-    def construct(self, **parameters: ArrayLike) -> Distribution:
+    def construct(self, **parameters: npt.ArrayLike) -> Distribution:
         """
         Create a distribution from an explicit set of parameters.
 
         Args:
-            **parameters (ArrayLike): Parameters that define a member of this family,
+            **parameters: Parameters that define a member of this family,
                 passed as sequential arguments.
 
         """
         return self._member(**parameters)
+
+    def sample(
+        self,
+        samples: int,
+        rng_key: jax.Array,
+        **kwargs: npt.ArrayLike,
+    ) -> npt.NDArray[float]:
+        """Sample values from the distribution."""
+        raise NotImplementedError
```

Comment:
```suggestion
```

Guessing this shouldn't be here (since this is the default "fall-back" method). Plus there's code underneath it which seems to do what we already do as a worst-case option.

Reply:
This was some debugging that I forgot to delete